code/modules/clothing/suits/armor.dm

```diff
@@ -51,9 +53,11 @@
 			add_fingerprint(user)
 			W.forceMove(src)
 			attached_badge = W
-
 			var/datum/action/A = new /datum/action/item_action/remove_badge(src)
 			A.Grant(user)
+			sprite_sheets = list(
+				"Grey" = 'icons/mob/species/grey/suit.dmi'
+			)
```

Comment:
Why does this need to be reassigned here in attackby? Is there a place where the armor gets its `sprite_sheets` value changed from the default, which already has this assigned?

Reply:
There's two armor sprites.
`icon_state = "armor"`
`icon_state = "armorsec"`

Reply:
Ah, my apologies, it was hard to see on Github. Looking into your suggestion offline.

Reply:
As expected, this works! :D
Nice catch! Thank you so much!